core/templates/domain/summary/learner-exploration-summary.model.ts

```diff
@@ -31,7 +31,7 @@ interface HumanReadableContributorsSummary {
 }
 
 export interface LearnerExplorationSummaryBackendDict {
-  'category': string;
+  'category'?: string | undefined;
```

Comment:
Why set both `?` and `undefined`? Also, where exactly can `category` be `undefined`?

Reply:
I'll change undefined to null. "?" is needed to make operand of delete operator optional on category.

Reply:
![image](https://user-images.githubusercontent.com/68547101/96965260-796d9800-1529-11eb-8d09-12c74e45dbf4.png)

Reply:
Remove the delete from the collection-node.model.spec.ts then, instead just change the category to a different string.

Reply:
Okay.